ingestion/functions/parsing/germany/germany.py

```diff
@@ -87,7 +87,10 @@ def parse_cases(raw_data_file: str, source_id: str, source_url: str):
                     "caseReference": {"sourceId": source_id, "sourceUrl": source_url},
                     "location": {
                         "query": ", ".join((row[_ADMIN2], row[_ADMIN1], "Germany")),
-                        "limitToResolution": "Admin2",
+                        # We could set "limitToResolution" to "Admin2" but even
+                        # though inputs are districts mapbox doesn't seem to have
+                        # relevant results when "district" restriction is
+                        # enforced so we do not set limitToResolution on purpose.
```

Comment:
Do you think we should pass `limitToResolution: "Admin2, Admin1, Country"`, or is it better to just elide the field? The impact of the former is avoiding too-fine-grained erroneous geocodes.

Reply:
right this feel safer thanks, done.